hack/defaults

```diff
@@ -59,6 +60,7 @@ function network_addons_sed {
     sed -i "s|value: .*sriov-device-plugin.*|value: ${NETWORK_ADDONS_SRIOV_DP_IMAGE}|g" ${TEMP_DIR}/operator.yaml
     sed -i "s|value: .*sriov-cni.*|value: ${NETWORK_ADDONS_SRIOV_CNI_IMAGE}|g" ${TEMP_DIR}/operator.yaml
     sed -i "s|value: .*mac-controller.*|value: ${NETWORK_ADDONS_KUBEMACPOOL_IMAGE}|g" ${TEMP_DIR}/operator.yaml
+    sed -i "s|value: sriov$|value: ${NETWORK_ADDONS_SRIOV_NETWORK_TYPE}|g" ${TEMP_DIR}/operator.yaml
```

Comment:
@phoracek I'm assuming you'll use the downstream value when you ship the network-operator?

Reply:
Yes, as Lukáš commented https://github.com/kubevirt/hyperconverged-cluster-operator/pull/38#pullrequestreview-228211842

Reply:
ok then `deploy/converged/operator.yaml` should change since it's downstream only
```
        - name: SRIOV_NETWORK_TYPE
          value: sriov
```

Reply:
Yes, we will need to add flags to manifests generator to cover all network addons images and this new variable.

Reply:
created an issue: #39